students/john_navitsky/session01/warmup1/missing_char.py

```diff
@@ -0,0 +1,9 @@
+def missing_char(word, index):
+    if word:
+        wordlength=len(word)
+        start=0
+        return word[start:index]+word[index+1:wordlength]
```

Comment:
you dont need start or wordlength here -- they can be assumed:

```
return word[:index]+word[index+1:]
```

pretty nifty, eh?